src/features/advanced/cxpo-auto-pricing.ts

```
@@ -0,0 +1,200 @@
+        return `${priceSell.value}`;
+      } else if (!showPriceSell.value) {
+        return `${priceBuy.value}`;
+      }
+      return 'Error';
+    });
+
+    watchEffectWhileNodeAlive(form, () => {
+      priceInput.placeholder = placeholderText.value;
+    });
+
+    const orderInfo = computed(() => cxobStore.getByTicker(tile.parameter!));
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)